### PR TITLE
adopt travis.yml to let it work as proposed in symfony best practices

### LINKED ISSUE #4: (Re-) enable travis as CI

### PR DESCRIPTION
**Pull request by ElectricMaxxx**
so i removed docker stuff as it should work on travis same. the travis script had been removed also, as it is recommended not to use them. hhvm was dropped by this change

fix #4

### CODE REVIEW

**Comment by ElectricMaxxx** [comment]
i added some more verbose output on our failing tests, to see thier issues. but i think there are two different ones
- str_repeat() - issue will end in a fix or adding in a conflict entry, when having a failing dependency
- timeout: tricky as there a command is tested and the test seems to be skipped in some situations and done in others.

**Comment by kunicmarko20** [comment]
When merging, could you do `squash and merge` so we get just 1 commit per PR

**Comment by kunicmarko20** [comment]
image

One of the reasons we should push from our forks 😄

**Comment by ElectricMaxxx** [comment]
Ok ... :-) i hope there are issues we can evaluate locally then.

**Comment by kunicmarko20** [comment]
Instead of creating PR with a branch from origin repo, you create PR from your fork branch, you get the same thing like now 😄

**Comment by kunicmarko20** [comment]
I mean, its not a big deal, if you like doing it this way, feel free to continue

File: tests/Command/CKEditorInstallerCommandTest.php [hunk 1]
```diff
@@ -48,6 +48,9 @@ protected function tearDown()
         }
     }
 
+    /**
+     * @group installation
```
[the remark this answers is not in the document]
Comment (kunicmarko20) [reply tests/Command/CKEditorInstallerCommandTest.php]: And I think this should be in a separated PR

**Pull request merged**